src/app/pages/core/not-found-page/not-found-page.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+
+import { SharedModule } from '@app/shared/shared.module';
+
+import { NotFoundPageComponent } from './not-found-page.component';
+
+@NgModule({
+  imports: [ SharedModule ],
+  declarations: [ NotFoundPageComponent ],
+})
+export class NotFoundPageModule { }
```

Comment:
how do you think about making it a `ErrorPagesModule` and place there all similar static error/information pages?

Reply:
Good idea, will do it.